### ISSUE TITLE
Limited Accessibility

### ISSUE THREAD

**Issue by calluna** [roots/soil#171]
## Submit a feature request
- [X ] This is a feature request

**What is the current behavior?**

No ARIA roles on the menu, and 'tab' does not work for dropdown

**What is the expected or desired behavior?**

the menu should be screen reader accessible and the user should be able to tab to the submenu items

---
Soil version: 3.7.1
WordPress version: 4.6.1

## Feature Request

Accessibility

**Comment by retlehs** [comment]
:+1: been meaning to add that, will take care of it soon

**Comment by calluna** [comment]
Tx! :)
